### PR TITLE
feat: add connectWallet and pending transactions

### PR DESCRIPTION
**Pull request by angelia-yuqi-personal**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/rigel/40)

### CODE REVIEW

File: src/components/WrapIcon/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import PropTypes from 'prop-types'
 import {CircleBg, SquareBg} from '../../assets/svg'
 
-function WrapIcon({type, size = 'w-4 h-4', children, className = ''}) {
+function WrapIcon({type, size = 'w-4 h-4', children, className = '', onClick}) {
   return (
     <div
+      onClick={() => onClick && onClick()}
+      aria-hidden={true}
```
Comment (zctocm) [inline src/components/WrapIcon/index.js:8]: aria-hidden="true"

File: src/pages/Shuttle/ShuttleForm/index.js [hunk 2]
```diff
@@ -74,7 +74,12 @@ function ShuttleForm() {
         {/* TODO SelectToken */}
         <div className="flex-1"></div>
       </div>
-      <WrapIcon type="circle" size="w-8 h-8" className="my-4">
+      <WrapIcon
+        type="circle"
+        size="w-8 h-8"
+        className="my-4"
+        onClick={invertChain}
```
Comment (zctocm) [inline src/pages/Shuttle/ShuttleForm/index.js:81]: The `invertChain` name should be `onInvertChain `

**Review by zctocm — APPROVED** [review]

File: src/components/WrapIcon/index.js (hunk 1 above)
Comment (angelia-yuqi-personal) [reply src/components/WrapIcon/index.js]: done

File: src/pages/Shuttle/ShuttleForm/index.js (hunk 2 above)
Comment (angelia-yuqi-personal) [reply src/pages/Shuttle/ShuttleForm/index.js]: done

**Review by zctocm — APPROVED** [review]

**Pull request merged**